recipes/r-rcppdpr/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set version = '0.1.10' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-rcppdpr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RcppDPR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RcppDPR/RcppDPR_{{ version }}.tar.gz
+  sha256: 92c91c2cdcf2030e0a1ffd7de23278b3cb6ab92c5a7bfe994b52bbc470bdb7d5
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
```

```suggestion
  number: 0
  rpaths:
    - lib/R/lib/
    - lib/
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
    - '*/Rblas.dll'       # [win]
    - '*/Rlapack.dll'     # [win]
```
We can add these lines to fix the linkage check on Windows.